Sources/Load.php

```diff
@@ -1280,10 +1280,10 @@ function loadMemberData($users, $is_name = false, $set = 'normal')
 		while ($row = $smcFunc['db_fetch_assoc']($request))
 		{
 			// If the image proxy is enabled, we still want the original URL when they're editing the profile...
-			$row['avatar_original'] = $row['avatar'];
+			$row['avatar_original'] = !empty($row['avatar']) ? $row['avatar'] : '';
 
 			// Take care of proxying avatar if required, do this here for maximum reach
-			if ($image_proxy_enabled && !empty($row['avatar']) && stripos($row['avatar'], 'http://') !== false)
+			if ((!empty($row['avatar'])) && $image_proxy_enabled && stripos($row['avatar'], 'http://') !== false)
```

Comment:
To ensure the column has a value prior to using stripos .. I did not see an error from that line but put the edit in to cover the possibility as I have seen undefined errors with my own projects on similar logic. The brackets could go around the first 2 variables or your other suggestion I also thought of prior can be implemented which will cover any null possibility.